### ISSUE TITLE
Docker: Gekko vs Japonicus containers?

### ISSUE THREAD

**Issue by rcoenen** [Gab0/japonicus#182]
I fundamentally fail to understand how the Docker setup for Gekko & Japonicus is supposed to work.

1) Gekko runs in its own container (`gekko_1`)
2) Japonicus runs in its own container (`japonicus_1`)
From how I understand Docker, this means that Japonicus' python code cannot access anything on the filesystem of the Gekko container and vice versa.

Yet Japonicus does try to access files on the Gekko filesystem:
for example in `japonicus/settings/_Global.toml `I am supposed to define `gekkoPath = '$HOME/gekko'`

However the Japonicus container has no access to te Gekko container and I get errors like:
```
japonicus_1  | Failure to find strategy parameter rules for RSI at ./strategy_parameters
japonicus_1  | Trying to locate strategy parameters at $HOME/gekko/config/strategies/RSI.toml
```

If I was not using Docker but simply installed gekko + japonicus on my local Mac file system then I could use absolute or relative paths and sure then Japnoicus could access Gekko's files.
But that cannot be done using Docker correct?

**Comment by Gab0** [comment]
japonicus originally acessed gekko's files, but then the docker images were created and we can't have it on docker. Simultaneously, I also realized it was a bad idea, so features accessing those files will be removed.
The `gekkoPath` thing can be safely ignored for now, and the strategy parameter search only goes to the gekko directory as a fallback from the main search path at `./strategy_parameters`. If you have the file at the main path the error goes away as it's required to have that `<strategy>.toml`file.
cheers!

**Comment by rcoenen** [comment]
Right. So the solution is: I need to copy all of Gekko's strategy files over to Japonicus?
Then that way it will find the (copy-) of the strategy and won't try to failback accessing the Gekko files (which are inaccessible from Japnocus' Docker anyway) - correct?

**Comment by Gab0** [comment]
Yeah, you could copy the gekko strategy parameters to japonicus, but the japonicus strategy params files are a bit different, check the examples at `strategy_parameters`.

Plz note that strategy parameters (`gekko/config/strategies`) are not gekko strategy files (`gekko/strategies`)... just in case XD

**Comment by rcoenen** [comment]
for Japonicus we're only interested in the .toml files with strategy settings correct?
the actual .js strategy files are to be executed by Gekko exclusively
